sundar2021reproducibility/openreview/packages.tex

```diff
@@ -0,0 +1,70 @@
+% COMMENT for Rescience-C
+% if you need to pass options to natbib, use, e.g.:
+%     \PassOptionsToPackage{numbers, compress}{natbib}
+% before loading neurips_2019
+
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[final]{neurips_2019} % Remove for ReScience
+
+% to avoid loading the natbib package, add option nonatbib:
+% \usepackage[nonatbib, final]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{caption}
+
+\DeclareCaptionLabelFormat{andfigure}{#1~#2  \&  \figurename~\thefigure}
+
+% Table and figure side by side
+\usepackage{tabularx}
+\usepackage[export]{adjustbox}
+
+\usepackage{multirow}
+\usepackage{comment}
+\usepackage{makecell}
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{amsmath}
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}      % images
+\usepackage[english]{babel}
```

Comment:
It seems this import is the issue from the logs: "! LaTeX Error: Option clash for package babel." Can you disable this and re-build?